neqo-transport/src/connection/tests/recovery.rs

```diff
@@ -294,21 +294,16 @@ fn pto_handshake_complete() {
     // Check that the other packets (pkt2, pkt3) are Handshake packets.
     // The server discarded the Handshake keys already, therefore they are dropped.
     // Note that these don't include 1-RTT packets, because 1-RTT isn't send on PTO.
-    let (pkt2_hs, pkt2_1rtt) = split_datagram(&pkt2.unwrap());
-    assert_handshake(&pkt2_hs);
-    assert!(pkt2_1rtt.is_some());
```

Comment:
If this is deliberate, the comment above needs updating, right?

> are Handshake packets.

Reply:
This comment remained unchanged since before #1603. So either it was already incorrect back then and we need to change it, or it should stay as is.